cmd/crio/main.go

```diff
@@ -353,6 +360,10 @@ func main() {
 			logrus.Fatalf("failed to listen: %v", err)
 		}
 
+		if len(config.Registries) == 0 {
+			logrus.Fatal("no registries configured. Please add registries with the --registry flag")
```

Comment:
Is this actually an error, or does it just force users to use fully qualified images